### PR TITLE
Clear potionEffects on Join and change swords emoji

### PR DESCRIPTION
**Pull request by mooziii**
Emoji works: 
grafik

### CODE REVIEW

File: src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/OnJoinManager.kt [hunk 1]
```diff
@@ -18,6 +18,10 @@ object OnJoinManager {
 
             val player = it.player
 
+            for (potionEffect in player.activePotionEffects) {
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/OnJoinManager.kt]: move this to `JoinPlayerReset`

**Review by jakobkmar — CHANGES_REQUESTED** [review]

**Comment by mooziii** [comment]
Moved

File: src/main/kotlin/net/axay/hglaborlobby/chat/ChatFormatter.kt [hunk 2]
```diff
@@ -26,7 +26,7 @@ object ChatFormatter {
         ":left:" to "←",
         ":right:" to "→",
         ":copyright:" to "©",
-        ":swords:" to "⚔",
+        ":sword:" to "🗡",
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/hglaborlobby/chat/ChatFormatter.kt:30]: keep original emoji "swords" and also add the new one under the name "sword" (don't replace the old one, just add the new one)

File: src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/OnJoinManager.kt [hunk 3]
```diff
@@ -18,6 +18,7 @@ object OnJoinManager {
 
             val player = it.player
 
+
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/OnJoinManager.kt]: ?

File: src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/JoinPlayerReset.kt [hunk 4]
```diff
@@ -10,6 +10,9 @@ object JoinPlayerReset {
 
         player.feedSaturate()
         player.heal()
+        for (potionEffect in player.activePotionEffects) {
```
Comment (jakobkmar) [inline src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/JoinPlayerReset.kt:13]: Are you sure, that this does not throw a `ConcurrentModificationException`?

**Review by jakobkmar — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/JoinPlayerReset.kt (hunk 4 above)
Comment (mooziii) [reply src/main/kotlin/net/axay/hglaborlobby/eventmanager/joinserver/JoinPlayerReset.kt:13]: Yes, I've used this a lot

**Review by jakobkmar — APPROVED** [review]

**Review by jakobkmar — APPROVED** [review]

**Pull request merged**